src/screens/Exchange/Exchange.js

```diff
@@ -443,6 +448,28 @@ class ExchangeScreen extends React.Component<Props, State> {
     }
   }
 
+  handleSellMax = () => {
+    const { value } = this.state;
+    const { balances } = this.props;
+    const selectedAssetSymbol = this.getSelectedFromAssetSymbol();
+    const chosenAssetBalance = formatAmount(getBalance(balances, selectedAssetSymbol));
+    value.fromInput.input = chosenAssetBalance;
```

Comment:
Hm, maybe it would be better to not mutate value here and instead make a copy of it first?